src/ghgvc_script.R

```diff
@@ -2,185 +2,94 @@
-    c(7L, 4L, 5L, 3L, 6L, 1L, 2L),
-    .Label = c("BR_soy", "BR_sugarcane", "miscanthus", "soybean", 
-               "Spring Wheat", "switchgrass", "US_corn"), class = "factor"), 
-  Rnet = c(0.01, 0.06, 0.02, 0.02, 0.05, 0.06, 0.17), 
-  latent = c(0.25, 0.26, 0.13, 0.24, 0.25, 0.1, 0.33)), 
-  .Names = c("ecosystem", "sw_radiative_forcing", "latent"), 
-  class = "data.frame", row.names = c(NA, -7L))
-
-sites <- names(config.list)[-which(names(config.list)%in% "options")]
-for (site in sites){
-  for(ecosystem.idx in which(names(config.list[[site]]) %in% "pft")){
-    ecosystem <- config.list[[site]][[ecosystem.idx]]$name
-    eidx <- biophys.defaults$ecosystem == ecosystem
-    if (any(eidx)){
-#      if(!exists(config.list[[site]][[ecosystem.idx]]$sw_radiative_forcing) | 
-#           config.list[[site]][[ecosystem.idx]]$sw_radiative_forcing %in% c(NA, 0)){
-        config.list[[site]][[ecosystem.idx]]$sw_radiative_forcing <- biophys.defaults$sw_radiative_forcing[eidx]
- #     }
- #     if(!exists(config.list[[site]][[ecosystem.idx]]$latent) | 
- #          config.list[[site]][[ecosystem.idx]]$latent %in% c(NA, 0)){
-        config.list[[site]][[ecosystem.idx]]$latent <- biophys.defaults$latent[eidx]
- #     }
-      
-    }
-  }
-} 
-
-
+###For each site, load ecosystem data
+#subtract options
+sites <- names(config.list)[-which(names(config.list) %in% "options")]
+
+########
+#Can remove if biophys defaults no longer necessary. All this block does is
+#Add biophys sw_radiative_forcing and latent defaults to the site ecosystems.
+# 
+# #### Hack to put in some defaults for biophysical variables
+# biophys.defaults <- structure(
+#   list(ecosystem = structure(
+#     c(7L, 4L, 5L, 3L, 6L, 1L, 2L),
+#     .Label = c("BR_soy", "BR_sugarcane", "miscanthus", "soybean", 
+#                "Spring Wheat", "switchgrass", "US_corn"), class = "factor"), 
+#     Rnet = c(0.01, 0.06, 0.02, 0.02, 0.05, 0.06, 0.17), 
+#     latent = c(0.25, 0.26, 0.13, 0.24, 0.25, 0.1, 0.33)), 
+#   .Names = c("ecosystem", "sw_radiative_forcing", "latent"), 
+#   class = "data.frame", row.names = c(NA, -7L))
+#
+#for (site in sites){
+#  for(ecosystem.idx in which(names(config.list[[site]]) %in% "pft")){
+#    ecosystem_name <- config.list[[site]][[ecosystem.idx]]$name
+#    
+#   eidx <- biophys.defaults$ecosystem == ecosystem
+#   if (any(eidx)){
+##      if(!exists(config.list[[site]][[ecosystem.idx]]$sw_radiative_forcing) | 
+##           config.list[[site]][[ecosystem.idx]]$sw_radiative_forcing %in% c(NA, 0)){
+#        config.list[[site]][[ecosystem.idx]]$sw_radiative_forcing <- biophys.defaults$sw_radiative_forcing[eidx]
+##     }
+##     if(!exists(config.list[[site]][[ecosystem.idx]]$latent) | 
+##          config.list[[site]][[ecosystem.idx]]$latent %in% c(NA, 0)){
+#        config.list[[site]][[ecosystem.idx]]$latent <- biophys.defaults$latent[eidx]
+##     }
+#      
+#    }
+#  }
+#} 
+
+### Run the GHG calculator V2
 x <- ghgvc2(config.list)
```

Comment:
note that `ghgvc2` is a hack function to clean up the output of `ghgvc`, but these should be merged as part of #3.